spec/models/medium_spec.rb

```diff
@@ -26,15 +30,17 @@
     end
 
     it 'should be valid with a valid content_type' do
```

Comment:
Do you wanna aggregate these failures too, since there's about 8 tests happening in here?